backend/package.json

```
@@ -33,6 +33,8 @@
     "@nestjs/config": "^4.0.2",
     "@nestjs/core": "^11.0.1",
     "@nestjs/event-emitter": "^3.0.1",
+    "@nestjs/jwt": "^11.0.2",
```

Biome (2.4.15)

[error] The key `@nestjs/jwt` was already declared.

(lint/suspicious/noDuplicateObjectKeys)